views/components/message.erb

```diff
@@ -1,3 +1,3 @@
-<p class="message message--<%= kind %>">
-    <%= message %>
-</p>
+<div class="message message--<%= kind %>" role="<%= kind == 'critical' ? 'alert' : 'status' %>">
```

Comment:
Moving the logic would be idea. I also have the component being used in `./views/options/book-this.erb`.

For this component, there are only two roles (`alert` and `status`) and technically four different states (`info`, `success`, `warning`, `critical`). The states follow what's mostly established within the Design System. The `role` tells screen readers how prominent the element should be communicated. The most annoying is `alert`, which is set to `critical` because it should really grab the person's attention.

Ideally, this should really be turned into a Design System web component. 🙃